apps/client-asset-sg/src/app/components/menu-bar-item/menu-bar-item.component.ts

```diff
@@ -0,0 +1,24 @@
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import { Component, HostBinding, Input } from '@angular/core';
+
+@Component({
+  selector: 'li[asset-sg-menu-bar-item]',
+  templateUrl: './menu-bar-item.component.html',
+  styleUrl: './menu-bar-item.component.scss',
+})
+export class MenuBarItemComponent {
+  @Input({ required: true })
+  icon!: string;
+
+  @Input()
+  link: string[] | null = null;
+
+  @Input({ transform: coerceBooleanProperty })
+  disabled = false;
+
+  @HostBinding('aria-disabled')
```

Comment:
Just a question for my understanding: Why use hostBInding and bind inside the tempalte?

Reply:
Host bindings add attributes to the component's host element (in this case, a `li`). That element is not defined in the component's template, but rather specified when the component is used (e.g. `<li asset-sg-menu-bar-item>...</li>`. There is no way to modify that element from within the component's template itself.

Reply:
I see, thanks for clarifying